dynamic-plugins/imports/package.json

```diff
@@ -16,23 +16,26 @@
     "tar": "6.2.1"
   },
   "peerDependencies": {
-    "@janus-idp/backstage-plugin-3scale-backend": "1.5.6",
-    "@janus-idp/backstage-plugin-aap-backend": "1.6.6",
-    "@janus-idp/backstage-plugin-acr": "1.3.6",
-    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.3.6",
-    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.3.6",
-    "@janus-idp/backstage-plugin-keycloak-backend": "1.9.6",
-    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.5.6",
-    "@janus-idp/backstage-plugin-ocm": "4.0.2",
-    "@janus-idp/backstage-plugin-ocm-backend": "4.0.2",
-    "@janus-idp/backstage-plugin-quay": "1.6.7",
-    "@janus-idp/backstage-plugin-rbac": "1.20.2",
-    "@janus-idp/backstage-plugin-tekton": "3.6.5",
-    "@janus-idp/backstage-plugin-topology": "1.19.5",
-    "@janus-idp/backstage-plugin-argocd": "1.0.4",
-    "@janus-idp/backstage-scaffolder-backend-module-quay": "1.4.6",
-    "@janus-idp/backstage-scaffolder-backend-module-regex": "1.4.6",
-    "@janus-idp/backstage-scaffolder-backend-module-servicenow": "1.4.7",
-    "@janus-idp/backstage-scaffolder-backend-module-sonarqube": "1.4.6"
+    "@janus-idp/backstage-plugin-3scale-backend": "1.5.15",
+    "@janus-idp/backstage-plugin-aap-backend": "1.6.15",
+    "@janus-idp/backstage-plugin-acr": "1.4.13",
+    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.9",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.11",
+    "@janus-idp/backstage-plugin-keycloak-backend": "1.9.12",
+    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.10",
+    "@janus-idp/backstage-plugin-ocm": "4.1.8",
+    "@janus-idp/backstage-plugin-ocm-backend": "4.0.8",
+    "@janus-idp/backstage-plugin-quay": "1.7.8",
+    "@janus-idp/backstage-plugin-rbac": "1.20.14",
+    "@janus-idp/backstage-plugin-tekton": "3.7.7",
+    "@janus-idp/backstage-plugin-topology": "1.21.10",
+    "@backstage-community/plugin-redhat-argocd": "1.6.1",
```

Comment:
the whole point of moving to an external repo like backstage/community-plugins and releasing under the @backstage-community scope is that you **cannot use a peer dependency** as it's no longer a peer of the janus-idp content. 

That's why we need a wrapper **instead**.